src/services/conversationsService.js

```diff
@@ -206,6 +206,20 @@ const clearConversationHistory = async function(token) {
 	return response
 }
 
+/**
+ * Set conversation as unread
+ *
+ * @param {string} token The token of the conversation to be set as unread
+ */
+const setConversationUnread = async function(token) {
+	try {
+		const response = axios.delete(generateOcsUrl('apps/spreed/api/v1/chat/{token}/read', { token }))
+		return response
+	} catch (error) {
```

Comment:
For me catching in the service is a real anti-pattern. at least when the endpoint can return 400 Bad Request etc.

Reply:
But we do it almost for every service in this file, what's the difference?

Reply:
We've been doing try and catch mostly in store and components, that's true

Reply:
Then i'd prefer to keep it consistent for now, and move discussion to the new issue

for conversationService: 14 / 26 requests use try/catch
for conversationStore: 7 / ..40 actions use try/catch